src/components/tx/TxAlerts.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState, useEffect } from 'react';
+
+
+const TxAlerts = (props) => {
+  const [tx, setTx] = useState(props.tx);
+
+  useEffect(() => {
+    setTx({context: 'This is the 1,000,000th transaction.'});
+  }, [props.tx]);
+
+  const defaultAlert = () => {
+    if (!tx.context) {
+      return null;
+    }
+
+    return (
+      <div className="alert alert-info backup-alert" role="alert">
```

Comment:
This `alert-info` or `alert-danger` could be defined depending on the metadata type as well.

Reply:
I've removed the alert